### ISSUE TITLE
Step::RebuildPackage & Step::TriggerServices do not create EventSubscription

### ISSUE THREAD

**Issue by hennevogel** [openSUSE/open-build-service#13535]
For both of those steps people assume status reports on the SCM but they don't create `EventSubscription`s if they are the only steps in the workflow.yml

**Comment by hennevogel** [comment]
Like 

```yml
push_workflow:
  steps:
    - rebuild_package:
        project: home:hennevogel:myfirstproject
        package: ctris
  filters:
    event: push
    branches:
      only:
        - master
```

**Comment by dmarcoux** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Step support/action matrix](https://trello.com/c/GbwYPuBv/207-step-support-action-matrix)

**Comment by jfehlig** [comment]
Happy new year! Any progress on this issue? If possible, I'd like to return to my workflow hacking during hackweek.

**Comment by hennevogel** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [SCM event/action Workflow::Step support table](https://trello.com/c/tTMcxchV/276-scm-event-action-workflowstep-support-table)

**Comment by hennevogel** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Hand reporting in rebuild_package/trigger_services steps](https://trello.com/c/ypR24apr/275-hand-reporting-in-rebuildpackage-triggerservices-steps)

**Comment by danidoni** [comment]
Fixed by #14666

**Comment by LebedevRI** [comment]
@danidoni thank you, this is awesome!

I would still urge y'all to reconsider #11522,
it may be very well possible that the behavior of `branch_package`
matches what happens on OBS when branching manually,
but the inability to automatically transfer the build matrix is,
at the very least, a significant missing UX feature,
that would be solvable with a new step, e.g. `copy_build_matrix`.
